lib/experimental/Layouts/StandardLayout/index.tsx

```diff
@@ -1,13 +1,27 @@
+import { cn } from "@/lib/utils"
+import { cva, VariantProps } from "class-variance-authority"
 import React from "react"
 
-export interface StandardLayoutProps {
-  children: React.ReactNode
+export interface StandardLayoutProps
+  extends VariantProps<typeof layoutVariants> {
+  children?: React.ReactNode
 }
 
-export function StandardLayout({ children }: StandardLayoutProps) {
+const layoutVariants = cva(
+  "relative flex w-full flex-1 flex-col gap-4 place-self-center overflow-y-auto px-6 py-5",
```

Comment:
why do we set `relative`?

Reply:
To kind of create a "box" nobody can escape, even when using `absolute` positioning. Isn't this correct?